tests/integration/test_integratord/test_integratord_read_json_file_deleted.py

```diff
@@ -0,0 +1,128 @@
+'''
+copyright: Copyright (C) 2015-2022, Wazuh Inc.
+           Created by Wazuh, Inc. <info@wazuh.com>.
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+brief: Integratord manages wazuh integrations with other applications such as Yara or Virustotal, by feeding
+the integrated aplications with the alerts located in alerts.json file. This test module aims to validate that
+given a specific alert, the expected response is recieved, depending if it is a valid/invalid json alert, an
+overlong alert (64kb+) or what happens when it cannot read the file because it is missing.
+components:
+    - integratord
+suite: integratord_read_json_alerts
```

Comment:
```suggestion
suite: integratord_read_json_file_deleted
```

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/pull/3163/commits/c6cf2eaa2eff9021bf970eaa888578eb779d0177